src/UI/MainWindow.vala

```diff
@@ -57,7 +57,7 @@ public class Spreadsheet.UI.MainWindow : ApplicationWindow {
                     if (cell != null) {
                         expression.text = cell.formula;
                         expression.sensitive = true;
-                        style_popup.add (new StyleModal (cell.style));
+                        style_popup.add (new StyleModal (cell.font_style, cell.cell_style));
```

Comment:
More as a note for the future: this should be run when loading a sheet too, otherwise you can't immediately edit the style of the cell selected by default.